lighthouse-core/test/fraggle-rock/config/config-test.js

```diff
@@ -5,11 +5,14 @@
  */
 'use strict';
 
-const BaseAudit = require('../../../audits/audit.js');
-const {nonSimulatedPassConfigOverrides} = require('../../../config/constants.js');
-const BaseGatherer = require('../../../fraggle-rock/gather/base-gatherer.js');
-const {initializeConfig} = require('../../../fraggle-rock/config/config.js');
-const {LH_ROOT} = require('../../../../root.js');
+import {jest} from '@jest/globals';
```

Comment:
this isn't injected but `describe`, `it`, etc is?

Reply:
correct.